tests/components/steam_online/__init__.py

```diff
@@ -62,16 +64,32 @@ def ISteamUser(self) -> None:
 
     def GetFriendList(self, steamid: str) -> dict:
         """Get friend list."""
-        return {"friendslist": {"friends": [{"steamid": ACCOUNT_2}]}}
-
-    def GetPlayerSummaries(self, steamids: str) -> dict:
+        fake_friends = []
+        for _i in range(0, 400):
+            fake_friends.append(
+                {"steamid": "".join(random.choices(string.digits, k=len(ACCOUNT_1)))}
+            )
+        return {"friendslist": {"friends": [{"steamid": ACCOUNT_2}, *fake_friends]}}
+
+    def GetPlayerSummaries(self, steamids: str | list[str]) -> dict:
         """Get player summaries."""
+        assert len(str(steamids).replace("'", "%27")) <= 8071
```

Comment:
I think I understand why you're doing this, but a comment would be helpful.

I think this is to make sure that post url encoding that it remains below a max character limit of the API. If so, using a url encode function and then storing the magic number as an constant would be helpful. Eg. `MAX_STEAM_IDS_SIZE = 8071`.

Reply:
I am not sure where to find the url encode function, tried searching in the codebase but couldnt find it used anywhere else

Reply:
It’s part of the standard library. Most API clients should be doing this type of validation, so you wouldn’t see it used here much. 

https://docs.python.org/3/library/urllib.parse.html#urllib.parse.urlencode